src/pyird/image/aptrace.py

```diff
@@ -43,7 +43,7 @@ def cross_section(dat, nrow, nap):
     return onerow_masked, peakind_cut
 
 
-def set_aperture(dat, cutrow, nap, plot=True):
+def set_aperture(dat, cutrow, nap, ign_ord=[], plot=True):
```

Comment:
To prevent to misinterpret the band, raise error when `len(ig_ord) >= 30`?
51-30 = 21
```python
if len(ig_ord) >= 30:
       raise ValueError("length of ign_ord should be < 30.")
```